device-types/Juniper/ACX5048-AC.yaml

```diff
@@ -0,0 +1,141 @@
+---
+manufacturer: Juniper
+model: ACX5048-AC
+slug: juniper-acx5048-ac
+part_number: ACX5048-AC
+u_height: 1
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

Reply:
Hey @danner26,

I have added the `airflow` attribute as requested.

As for the device weight, the datasheet only provides a "loaded" weight with all module bays populated. Please let me know if you would like to include this, however I am of the understanding that a true device weight and weight for each individual module would be more accurate and therefore preferred.

Cheers,
Louis

Reply:
Hello, thanks for getting that! No we would want the unloaded weight so you can omit this